.circleci/config.yml

```
@@ -619,6 +729,11 @@ workflows:
             tags:
               only: /.*/
 
+      - build-docker-admin:
+          filters:
+            tags:
+              only: /.*/      
```

There are tailing spaces here.
```suggestion
              only: /.*/
```